ccd-definition/SearchResultFields.json

```diff
@@ -9,22 +9,29 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "caseIDReference",
-    "Label": "Case ID",
+    "CaseFieldID": "familyManCaseNumber",
+    "Label": "FamilyMan case number",
     "DisplayOrder": 2
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "caseLocalAuthority",
-    "Label": "Local authority",
+    "CaseFieldID": "[STATE]",
+    "Label": "State",
     "DisplayOrder": 3
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "[CREATED_DATE]",
-    "Label": "Case creation date",
+    "CaseFieldID": "caseLocalAuthority",
```

Comment:
Is that requirement to have different fields ordering? If not I suggest having one order of search fields so user does not have to learn location every time.